README.md

```diff
@@ -203,6 +203,14 @@ import { application } from "controllers/application";
 start({ application });
 ```
 
+## Testing & Demo
+
+Automatic testing of this library is done through a demo _Rails_ application. This application is in the `demo` folder. The tests are in `demo/spec`.
```

Comment:
the specs for the gem should be in the `/spec` folder, not in the demo app. You can have a look at devise if you need some inspiration.

Reply:
They were at first, but then the system tests were failing and I was not able to fix it. I'll give it a new try.

Reply:
Well, I'll be damned. Now it works.
This is done a way better looking ✅